packages/cli/core/src/commands/ui/create/__snapshots__/vue.spec.ts.snap

```diff
@@ -0,0 +1,75 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ui:create:vue when a directory with the projectName exists when the directory is empty should use the dir and init the project 1`] = `
+[
+  "npm",
+  [
+    "init",
+    "@coveo/headless-vue@1.0.0",
```

Comment:
I think you can use [Jest  property matcher](https://jestjs.io/docs/snapshot-testing#property-matchers) to not hard code the version in the snapshot

Reply:
Ya right, but well, it become hen easier to proceed with a expect.tobeNthcalledwith. so that's why I did